bofire/data_models/features/continuous.py

```diff
@@ -65,7 +65,9 @@ def round(self, values: pd.Series) -> pd.Series:
         allowed_values = np.arange(
             self.lower_bound, self.upper_bound + self.stepsize, self.stepsize
         )
-        idx = abs(values.values.reshape([3, 1]) - allowed_values).argmin(axis=1)  # type: ignore
+        idx = abs(values.values.reshape([len(values), 1]) - allowed_values).argmin(
+            axis=1
+        )  # type: ignore
```

Comment:
Can you double check if this `type: ignore` is really necessary?

Reply:
Good catch. It is not needed!